data.sql

```diff
@@ -29,3 +29,52 @@ VALUES (10, 'Blossom', '1998-10-13', 3, true, 17.0);
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutred, weight_kg)
 VALUES (11, 'Ditto', '2022-05-14', 4, true, 22.0);
 
+
+INSERT INTO owners(id, full_name, age) VALUES(1, 'Sam Smith', 34);
+INSERT INTO owners(id, full_name, age) VALUES(2, 'Jennifer Orwel', 19);
+INSERT INTO owners(id, full_name, age) VALUES(3, 'Bob', 45);
+INSERT INTO owners(id, full_name, age) VALUES(4, 'Melody Pond', 77);
+INSERT INTO owners(id, full_name, age) VALUES(5, 'Dean Winchester', 14);
+INSERT INTO owners(id, full_name, age) VALUES(6, 'Jodie Whittaker', 38);
+
+INSERT INTO species(name, id) VALUES('pokemon', 1);
+INSERT INTO species(name, id) VALUES('digimon', 2);
+
+UPDATE animals  SET species_id = 2 WHERE name LIKE '%mon%';
+UPDATE animals SET species_id = 1 WHERE species_id IS NULL;
+
+UPDATE animals SET owners_id = 1 WHERE name = 'Agumon';
+UPDATE animals SET owners_id = 2 WHERE name = 'Gabumon' OR name = 'Pikachu';
+UPDATE animals SET owners_id = 3 WHERE name = 'Devimon' OR name = 'Plantmon';
+UPDATE animals SET owners_id = 4 WHERE name = 'Charmander' OR name = 'Squirtle' OR name = 'Blossom';
+UPDATE animals SET owners_id = 5 WHERE name = 'Angemon' OR name = 'Boarmon';
+
+
+INSERT INTO vets(name, age, date_of_graduation)
+VALUES('William Tatcher', 45, '2020-04-23'),
+('Maisy Smith', 26, '2019-01-17'),
+('Stephanie Mendez', 64, '1981-05-04'),
+('Jack Harkness', 38, '2008-06-08');
+
+INSERT INTO specializations(vets_id, species_id)
+VALUES(1, 1),(2, 1), (4, 2);
+
+INSERT INTO visits(vets_id, animals_id, visit_date)
+VALUES(1, 1, '2020-05-24'),
+(3, 1, '2020-07-22'),
+(4, 2, '2021-02-02'),
+(2, 3, '2020-06-05'),
+(2, 3, '2020-03-08'),
+(2, 3, '2020-05-14'),
+(3, 4, '2021-05-04'),
+(4, 5, '2021-02-04'),
+(2, 7, '2019-12-21'),
+(1, 7, '2020-08-10'),
+(2, 7, '2021-04-07'),
+(3, 6, '2019-09-29'),
+(4, 1, '2020-10-03'),
+(4, 1, '2020-11-04'),
+(2, 9, '2019-01-24'),
+(2, 9, '2019-05-15'),
+(3, 10, '2020-05-24'),
+(1, 10, '2021-01-11');
```

Comment:
![image](https://github.com/rachaouldbabaali/vet-clinic-db/assets/93039359/a4970d4b-4d86-41b3-ac35-1d1706060af2)

- [ ] It would be nice if you double check the requirements and see how many queries are in the project requirments and how many are there in your project because in requirements there are 20 but in your project there are 18 😨 Kindly add the missing entries to make your database has the exact same data which your project requirements demand 👍
- [ ] It would be nice after making all the requirements if you provide the new screenshots and replace them with the old screenshots in the pull request summary so the next reviewer will be able to verify if your database is upto your project requirements 😊